src/haddock/modules/analysis/seletopclusts/defaults.yaml

```diff
@@ -18,3 +18,15 @@ top_models:
   long: Number of best-ranked models to select per cluster. By default, 10 models are selected. If one expects to find many clusters (e.g., in the case of vague restraints), this number can be reduced to speed up the protocol. Instead, if few clusters are expected (very specific and localized restraints) this number can be increased.
   group: analysis
   explevel: easy
+offline:
+  default: false
+  type: boolean
+  title: Remove any web calls.
+  short: Completely isolate the haddock3 run & results from internet.
```

Comment:
"Remove any web calls."  does not make sense.

Also how does this influence the usage of the haddock3 webapp cdn? ping @sverhoeven

Reply:
was already discussed with @sverhoeven, there is no issue.